instana-java-sdk/src/main/java/com/instana/sdk/support/SpanSupport.java

```diff
@@ -261,27 +324,35 @@ public static void inheritNext(String traceId, String spanId) {
   }
 
   /**
-   * Clears an open span, i.e. ends a currently open span without committing it.
+   * Clears the currently open SDK span, i.e. ends the currently open span without committing it.
```

Comment:
- We never specify what an SDK span is
- It is not clear this method does not affect AutoTrace spans
- We should clarify we will clear the most recent SDK span.
- What happens if ENTRY is an SDK span, but it has AutoTrace children?